### PR TITLE
Updates upstream to v1.0.18

### LINKED ISSUE #4: Update to v1.0.18
Released May 31st: https://github.com/jedisct1/libsodium/releases/tag/1.0.18-RELEASE
Required for Schnorr signatures

### PR DESCRIPTION
**Pull request by denizzzka**
Fixes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,9 +25,11 @@ The bindings were generated with the following procedure:
 - Checkout the required version
 - Translate C headers to D modules:
 ```sh
-find src/libsodium/include/sodium -depth 1 -name "*.h" | xargs -I F $DSTEP -Isrc/libsodium/include/sodium/ F -o $LIBSODIUMD/source/libsodium/$(basename F | cut -d'.' -f1).d
+find "$LIPSODIUM_REPO/src/libsodium/include/sodium" -maxdepth 1 -xtype f -name '*.h' \
```
Comment (Geod24) [inline README.md]: `LIBSODIUM_REPO`

File: source/libsodium/package.d [hunk 2]
```diff
@@ -22,6 +22,7 @@ public import libsodium.crypto_box_curve25519xsalsa20poly1305;
 public import libsodium.crypto_core_ed25519;
 public import libsodium.crypto_core_hchacha20;
 public import libsodium.crypto_core_hsalsa20;
+public import libsodium.crypto_core_ristretto255;
```
Comment (Geod24) [inline source/libsodium/package.d:25]: Please squash this commit in the one that introduce those modules

File: README.md (hunk 1 above)
Comment (denizzzka) [reply README.md]: I added LIBSODIUM_REPO description into below

Comment (Geod24) [reply README.md]: I mean, there's a typo. You wrote `LIPSODIUM_REPO`

Comment (denizzzka) [reply README.md]: Ah, got it)

**Review by Geod24 — APPROVED** [review]

**Pull request merged**